src/controllers/user.controllers.js

```diff
@@ -32,17 +33,40 @@ export const userLogin = async (data, response) => {
   const { error } = loginValidation(data);
   if (error) return response.status(StatusCodes.BAD_REQUEST).send(error.details[0].message);
 
-  // const user = await User.findOne({ email: data.email });
-  const user = await User.findOne({ email: 'HasloToTest123@gmail.com' });
-  if (!user) return response.status(StatusCodes.BAD_REQUEST).send('Email or password is wrong');
+  // const users = await User.find({ email: data.email });
+  const users = await User.find({ email: 'HasloToTest123@gmail.com' }); // Dodano na czas testowania
+  const activeUser = users.filter((user) => {
+    if (user.statusUser == 1) {
+      return user;
+    }
```

Comment:
we can add else{    return response.status(StatusCodes.BAD_REQUEST).send('Email or password is wrong');}

instead line 44 and 45